lotus/client.go

```diff
@@ -5,12 +5,10 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/filecoin-project/lotus/api/apistruct"
-	"github.com/filecoin-project/lotus/build"
+	"github.com/filecoin-project/lotus/lib/jsonrpc"
```

Comment:
Notice we use directly the `jsonrpc` impl of Lotus and not having our own (based on them). So one less thing to mantain.

Reply:
That, plus not having to maintain all those API structs is much cleaner!